### PR TITLE
navigation between  pages

### PR DESCRIPTION
**Pull request by voronevska**

### CODE REVIEW

File: src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { LinkItem } from '../LinkItem';
+
+export const Navigation = () => {
+  const links = [
+    { to: '/', label: 'Home' },
+    { to: '/phones', label: 'Phones' },
+    { to: '/tablets', label: 'Tablets' },
+    { to: '/accessories', label: 'Accessories' },
+  ];
```
Comment (PashaMalyshkin) [inline src/components/Navigation/Navigation.tsx:9]: replace to constants file

File: src/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <nav className="nav">
+      <ul className="nav__list">
+        {links.map(({ to, label }) => (
+          <LinkItem
+            key={to}
+            to={to}
+            label={label}
+          />
+        ))}
+      </ul>
```
Comment (PashaMalyshkin) [inline src/components/Navigation/Navigation.tsx:21]: Create Links List separate component

File: src/components/LinkItem/LinkItem.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+export const LinkItem: FC<Props> = ({ to, label }) => {
+  return (
+    <li className="nav__item">
+      <Link
+        to={to}
+        className={cn('nav__link', {
+          'nav__link--active': useLocation().pathname === to,
+        })}
+      >
```
Comment (PashaMalyshkin) [inline src/components/LinkItem/LinkItem.tsx:18]: You can use NavLink, where you can use func in className

File: src/App.tsx [hunk 4]
```diff
@@ -1,13 +1,22 @@
...
 import { NotFoundPage } from './pages/NotFoundPage';
-import { PhonePage } from './pages/PhonesPage';
 import { Footer } from './components/Footer';
 
 const App = () => (
   <>
+    <Helmet>
+      <body className="store-theme store-theme--dark" />
+    </Helmet>
+
```
Comment (PashaMalyshkin) [inline src/App.tsx:19]: Better use useContext for theme

File: src/styles/blocks/icon-bar.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  &__icon {
+    margin: 16px;
+    height: 16px;
+    fill: $theme-color-light;
+
+    @include onTablet {
+      margin: 24px;
+    }
+  }
+}
```
Comment (PashaMalyshkin) [inline src/styles/blocks/icon-bar.scss:44]: Put components into an src/components folder.
each component should be a folder with index.ts, ComponentName.tsx, ComponentName.module.scss files.
use CSS modules.
keep .module.scss files together with their components.

**Review by PashaMalyshkin — CHANGES_REQUESTED** [review]

**Review by Esteban4769 — COMMENTED** [review]
Good job. Should probably rename all sccs modules like <module_name>.sccs instead of  <module_name>.module.sccs and remove LinkItem copy

**Pull request closed without merging**